crates/tabby-scheduler/queries/kotlin.scm

```diff
@@ -0,0 +1,19 @@
+(
+  (function_declaration (simple_identifier) @name) @definition.function
+)
+
+(
+  (class_declaration (type_identifier) @name) @definition.class
+)
+
+(
+  (object_literal  (delegation_specifier) @name) @definition.object
+)
```

Comment:
```suggestion
```

is object literal something actually referencable, from a cross file context perspective?

Reply:
yes it is, I woud keep it